packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -4,6 +4,7 @@ import { OptionsWithTooltip, TooltipDisplayMode, SortOrder } from '@grafana/sche
 export function addTooltipOptions<T extends OptionsWithTooltip>(
   builder: PanelOptionsEditorBuilder<T>,
   singleOnly = false,
+  setProximity = false,
```

Comment:
this is called a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) :)

you end up with `addTooltipOptions(builder, false, true, opts);`, which is a confusing API

not worth cleaning it up in this PR, but might consider switching to a more explicit props object in the future.